.pre-commit-config.yaml

```diff
@@ -86,12 +86,8 @@ repos:
         types: [python]
       - id: no-unseeded-default-rng
         name: no-unseeded-default-rng
-        description: 'Enforce that no non-seeded default_rng is used and default_rng is used instead of np.random.seed'
-        entry: |
-          # Check for usage of default_rng without seeding
-          default_rng\(\)|
-          # Check for usage of np.random.seed
-          np.random.seed\(
```

Comment:
Should we keep this without the parenthesis to forbid usage of np.random.seed in favor of the newer default_rng API?

Reply:
Ah interesting, NPY002 allows the `np.random.seed` callable to exist but complains when it's called. Sure I'll add this back.